### PR TITLE
 Add more italian data

### PR DESCRIPTION
**Pull request by miscelatore**
Hi, I re-submit my changes concerning the support for the Italian language of some modules. 
I fix the gender_pl.rb file.

Regards. 
Stefano

### CODE REVIEW

File: test/test_units.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require_relative 'helper'
+require 'ostruct'
```
Comment (Volosh1n) [inline test/test_units.rb:4]: Just curious: what the purpose of this change? Other changes seems good to me

**Review by Volosh1n — APPROVED** [review]

**Comment by miscelatore** [comment]
I run tests, some of them fail on test / test_units.rb.
After add
require 'ostruct'
100% test ok

I used ruby 2.6.1p33.



Il giorno lun 9 ago 2021 alle ore 09:21 Vsevolod Voloshyn <
***@***.***> ha scritto:

**Comment by miscelatore** [comment]
Additional note.
With version 2.6.6 of ruby, the require 'ostruct'
it is not required to pass the tests successfully.

Il giorno lun 9 ago 2021 alle ore 09:36 Stefano Ruberti <
***@***.***> ha scritto:

File: test/test_units.rb (hunk 1 above)
Comment (miscelatore) [reply test/test_units.rb]: I run tests, some of them fail on test / test_units.rb.
After add 
require 'ostruct' 
100% test  ok
I used ruby 2.6.1p33.

Additional note.
With version 2.6.6 of ruby, the require 'ostruct'
it is not required to pass the tests successfully.

File: lib/ffaker/address_it.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module FFaker
+  module AddressIT
+    include FFaker::Address
```
Comment (Volosh1n) [inline lib/ffaker/address_it.rb:5]: Please, check if this file works without including `FFaker::Address`

Comment (miscelatore) [reply lib/ffaker/address_it.rb:5]: Yes, I confirm thats work without including FFaker::Address.
However, in the other modules the FFaker :: Address of the other languages ​​is still present

Comment (Volosh1n) [reply lib/ffaker/address_it.rb:5]: Yes, we will remove it where it is not necessary. Thank you

Comment (miscelatore) [reply lib/ffaker/address_it.rb]: I did other tests. Using ruby ​​2.6.6 and 2.7.2 you also need to add require 'set' in test_array_utils.rb to get 100% test ok

Comment (Volosh1n) [reply lib/ffaker/address_it.rb]: So... I see you returned back `require 'ostruct'`

File: lib/ffaker/address_it.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module FFaker
+  module AddressIT
+    #include FFaker::Address
```
Comment (Volosh1n) [inline lib/ffaker/address_it.rb:5]: could you, please, remove it?

Comment (miscelatore) [reply lib/ffaker/address_it.rb]: Done.

File: test/test_units.rb (hunk 1 above)
Comment (Volosh1n) [reply test/test_units.rb:4]: So, let's remove it?

File: lib/ffaker/address_it.rb (hunk 3 above)
Comment (Volosh1n) [reply lib/ffaker/address_it.rb]: Thank you!

File: lib/ffaker/address_it.rb (hunk 2 above)
Comment (miscelatore) [reply lib/ffaker/address_it.rb]: Using ruby ​​2.6.6 test fails if require 'ostruct' is not defined

**Comment by Volosh1n** [comment]
@miscelatore Released in [2.19.0](https://github.com/ffaker/ffaker/blob/v2.19.0/Changelog.md#2190)

**Comment by miscelatore** [comment]
I've explain this before in a previous my comment, it's necessary to pass
some test on my rails environment

Bye

Il lun 9 ago 2021, 09:21 Vsevolod Voloshyn ***@***.***> ha
scritto:

**Pull request merged**